index.html

```diff
@@ -84,67 +84,89 @@ <h3>Check out where<br> others have gone</h3>
         </div>
     </div>
 
-       
+
```

Comment:
line 88 to 94 is duplicate code (and an error since one only has a class and not an attribute in it). could you delete those parts?

Reply:
Removed duplicate code